lib/compiler-finder.js

```diff
@@ -160,7 +160,7 @@ class CompilerFinder {
                 return [];
             } else {
                 let arr = [];
-                for (const el of envVarsString.split(':')) {
+                for (const el of envVarsString.split(';')) {
```

Comment:
I know this change makes a lot of sense, but this makes me curious about what's in the envVars for MSVC (https://github.com/mattgodbolt/compiler-explorer/blob/master/lib/compilers/win32.js#L105) and if this doesn't break anything. @ubsan do you know?

Reply:
I can only presume some local changes needed full paths (MS compilers often have paths in the environment variables, as I remember). This is a shortcoming of our configuration system :*(